### PR TITLE
reconfigure images for 1.11

### PR DESCRIPTION
**Pull request by BenTheElder**
doing some fixup and upgrades here
- [x] add 1.11 branch to the kubekins-e2e image
- [x] drop 1.7 variant for kubekins-e2e
- [x] update `experiment/bump_e2e_image.sh` to drop 1.7 and add 1.11
- [x] upgrade to bazel 0.14 for master and 1.11 branches
- [x] bump images
- [x] switch 1.11 jobs to 1.11 image
- [x] reconcile with https://github.com/kubernetes/test-infra/pull/8256 ?

followup:
- we should fork the presubmits to the 1.11 image, but that diff is probably enough to warrant a separate PR, and is also less critical since there can't be 1.11 presubmits yet.

### CODE REVIEW

**Comment by BenTheElder** [comment]
fixes https://github.com/kubernetes/test-infra/pull/8260
will rebase on top of https://github.com/kubernetes/test-infra/pull/8256 before bumping images

**Comment by BenTheElder** [comment]
/hold
need to look for a test run of the new image

**Comment by BenTheElder** [comment]
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-canary/16995?log#log
green with 
> I0605 18:28:50.713] Image: gcr.io/k8s-testimages/kubekins-e2e:v20180605-65a672b59-master

**Comment by BenTheElder** [comment]
fixes https://github.com/kubernetes/test-infra/issues/8258

PTAL

File: prow/config.yaml [hunk 1]
```diff
@@ -12095,451 +12095,451 @@ periodics:
...
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20180605-65a672b59-master
 - tags:
   - generated   # AUTO-GENERATED by experiment/generate_tests.py - DO NOT EDIT!
   interval: 1h
   agent: kubernetes
   labels:
-    preset-service-account: "true"
-    preset-k8s-ssh: "true"
+    preset-service-account: 'true'
+    preset-k8s-ssh: 'true'
```
Comment (krzyzacy) [inline prow/config.yaml]: /shrug

I didn't find an option for use single/double quotes..

Comment (BenTheElder) [reply prow/config.yaml]: this is auto generated AFAIK 😱 
seems to work though?

Comment (krzyzacy) [reply prow/config.yaml]: I mean it's probably from one of the yaml.dump

this is fine?

Comment (BenTheElder) [reply prow/config.yaml]: http://yaml.org/spec/1.2/spec.html#id2788097
single quotes are definitely valid

Comment (krzyzacy) [reply prow/config.yaml]: I mean if we should unify everything to single quote..

**Pull request merged**